content/d-sgov/d-sgov-sbírka/d-sgov-sbírka-glosář.ttl

```diff
@@ -440,3 +498,549 @@ d-sgov-sbírka-pojem:url-cíle-odkazu a skos:Concept;
 d-sgov-sbírka-pojem:kotva-cíle-odkazu a skos:Concept;
   skos:inScheme d-sgov-sbírka:glosář;
   skos:prefLabel "kotva cíle odkazu"@cs .
+
+d-sgov-sbírka-pojem:nález-rozhodnutí-v-bloku a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Nález rozhodnutí v bloku"@cs .
+
+d-sgov-sbírka-pojem:citace-v-bloku a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Citace v bloku"@cs .
+
+d-sgov-sbírka-pojem:nález-odůvodnění-v-bloku a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Nález odůvodnění v bloku"@cs .
+
+d-sgov-sbírka-pojem:vnořená-úroveň-v-bloku a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Vnořená úroveň v bloku"@cs .
+
+d-sgov-sbírka-pojem:novelizační-blok a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Novelizační blok"@cs .
+
+d-sgov-sbírka-pojem:novelizační-bod a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Novelizační bod"@cs .
+
+d-sgov-sbírka-pojem:přechodné-ustanovení-v-bloku a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Přechodné ustanovení v bloku"@cs .
+
+d-sgov-sbírka-pojem:příloha-v-bloku a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Příloha v bloku"@cs .
+
+d-sgov-sbírka-pojem:závěrečná-ustanovení-v-bloku a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Závěrečná ustanovení v bloku"@cs .
+
+d-sgov-sbírka-pojem:bod a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Bod"@cs .
+
+d-sgov-sbírka-pojem:odrážka a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Odrážka"@cs .
+
+d-sgov-sbírka-pojem:článek a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Článek"@cs .
+
+d-sgov-sbírka-pojem:článek-textového-typu a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Článek textového typu"@cs .
+
+d-sgov-sbírka-pojem:díl a skos:Concept;
+  skos:inScheme d-sgov-sbírka:glosář;
+  skos:prefLabel "Díl"@cs .
+
+d-sgov-sbírka-pojem:fs a skos:Concept;
```

Comment:
Maybe we should check with someone with legal background which categories make sense to materialize and which not. @karelklima?

(Maybe @MichalMed you already did it I just wasn't sure)

Reply:
Nedělal jsem to, bral jsem to jako součást "kontroly"

Reply:
Pokud to chápu správně - jde o to rozhodnout, které typy fragmentů zahrnout do modelu (vs. použít nějaký generický fragment)?

Většina typů vypadá rozumně, ale některé jsou evidentně čistě technické. Třeba "FS" = to netuším, co je. Stejně tak různé typy odstavců, bodů, ... Vidím tady taky reálné riziko, že některé pojmy mohou být ambivalentní a nabývat různého významu v závislosti na typu právního aktu. Ale to bez většího vzorku dat stejně neověříme.

Reply:
Ano, je to tak. Ty technické bychom do modelu dávat neměli. Jen si nejsem jistý, co všechno je technický pojem a co ne.

Reply:
Souhlasím, že technické ne, jen bych upozornil že ty FS apod. jsou tak trochu deprecated. Začal jsem s přepisem hodnot a cca v třetině jsem se na to vykašlal a nechal si to tam pro případ, že bych je později potřeboval. Nepotřebuju, smažu, pošlu nový PR. Debata pouze prosím jen k těm pojmům, které zůstanou (jsou to ty v tabu Číselníky v OG, jestli tedy funguje správně vizualizace pojmů).